### DISCUSSION TITLE
2nd engine binding

### DISCUSSION THREAD

**Discussion by CJORDAN371** [flybywiresim/aircraft#5303] [General]
The 2nd engine doesn't spool up when the throttle is advanced, it only spools up when the 1st engine is spooled up. 
the little green marker shows where the throttle position is on the 2nd engine and it works. I've tried both with binding the throttles using the sim control binding and FSUIPC with the same results. not a big problem unless you are trying to make a left turn by increasing #2 engine.

**Comment by hotshotp** [comment]
please calibrate throttles in the EFB, it is a requirement

**Comment by tracernz** [comment]
Are you on stable? Disengage autothrottle. The MSFS autothrottle impl. controls both off the one input. Alternately you can move to dev version, calibrate your throttles in the EFB and autothrottle will work correctly.
